core/eth/state.go

```diff
@@ -28,7 +28,14 @@ func (cs *ChainState) GetOperatorStateByOperator(ctx context.Context, blockNumbe
 		return nil, err
```

Comment:
can we move these methods to eth reader and remove this struct? 
Not sure why we need a separate struct for this

Reply:
If we're caching the operator sockets, maybe we should keep this struct separate